### PR TITLE
Add banner to scheduled workflows, fix link

### PR DESCRIPTION
**Pull request by devinmarieb**
# Description
Add banner to scheduled workflows to indicate sunsetting date.
Fix link in server docs.

# Reasons
A duplicate PR as the other one was stuck in review for over 20 days waiting on approval from a different team.

### CODE REVIEW

File: jekyll/_cci2/v.2.17-overview.md
Comment (thomasr8) [inline jekyll/_cci2/v.2.17-overview.md:114]: @devinmarieb  Should this be "roll back" instead of "rollback"?

**Review by thomasr8 — APPROVED** [review]
@devinmarieb Looks good...just one very minor question.

**Comment by devinmarieb** [comment]
@circleci/docs-disco Not really sure how or why the two commits ended up here -- I needed to merge with master to be able to get this to pass from the previous pipeline errors. Feel free to not approve this if I need to get those two commits out of here.

**Pull request closed without merging**
